app/controllers/customers_controller.rb

```diff
@@ -279,8 +279,11 @@ def auto_complete_for_customer
     @customers_s =
       Customer.find_by_multiple_terms(s.split( /\s+/)).
           reject {|customer| @customers.include?(customer)}
-    result = @customers.map do |c|
-      {'label' => c.full_name, 'value' => customer_path(c)}
+    session[:customers_filter] = params[:term]
+    result = Array[{'label' => 'show all',
```

Comment:
Is it duplicate to use Array() and [ ] to initialize at the same time?

Reply:
From my perspective, they are the same.